namui-cli/electron/src/namuiApi/fileSystem/read.js

```diff
@@ -0,0 +1,11 @@
+const { readFile } = require("fs/promises");
+const { join } = require("path");
+
+const resourceRoot = join(__dirname, "../../../..");
```

Comment:
If we build on electron target, It we be placed at `wasm_linux_x64_electron/resources/app/src/namuiApi/fileSystem/read.js`. 
So `resourceRoot` means `wasm_linux_x64_electron/resources`